relayer/chains/cosmos/grpc_query.go

```diff
@@ -148,13 +152,49 @@ func (cc *CosmosProvider) TxServiceBroadcast(ctx context.Context, req *tx.Broadc
 		return nil, status.Error(codes.InvalidArgument, "invalid empty tx")
 	}
 
-	resp, err := cc.BroadcastTx(ctx, req.TxBytes)
-	if err != nil {
+	var (
+		blockTimeout = defaultBroadcastWaitTimeout
+		err          error
+		rlyResp      *provider.RelayerTxResponse
+		callbackErr  error
+		wg           sync.WaitGroup
+	)
+
+	if cc.PCfg.BlockTimeout != "" {
+		blockTimeout, err = time.ParseDuration(cc.PCfg.BlockTimeout)
+		if err != nil {
+			// Did you call Validate() method on ChainClientConfig struct
```

Comment:
This is an artifact from `lens`, `ChainClientConfig` is actually `CosmosProviderConfig` now